### PR TITLE
[docs] Update the contribution guide to reflect changes in the docs workflow

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs @Nuarat @miherlosev 

Updated the contribution guide with details on how to contribute to documentation.

### CODE REVIEW

**Comment by VasilyStrelyaev** [comment]
Let's show it to Dirk before we merge

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+* [Contribute to Documentation](#contribute-to-documentation)
+  * [Documentation for the Current Functionality](#documentation-for-the-current-functionality)
+  * [Documentation for New Features](#documentation-for-new-features)
 
 ## Code of Conduct
 
-TestCafe has adopted a [Contributor Code of Conduct](CODE_OF_CONDUCT.md), abide by its terms.
+TestCafe abides by the [Contributor Code of Conduct](CODE_OF_CONDUCT.md).
 
 ## General Discussion
```
Comment (dirk-pieterse) [inline CONTRIBUTING.md]: "If you find a problem" Why would you 'find' a problem?

Comment (dirk-pieterse) [reply CONTRIBUTING.md]: "When you create a new issue" create an issue? Why would you create an issue?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,33 +1,35 @@
...
 
 Join the TestCafe community on Stack Overflow: ask and answer [questions with the TestCafe tag](https://stackoverflow.com/questions/tagged/testcafe).
 
 ## Reporting a Problem
 
-If you find a problem when using TestCafe, please file an issue in our [GitHub repository](https://github.com/DevExpress/testcafe/issues).
+If you encounter a bug when using TestCafe, please file an issue in our [GitHub repository](https://github.com/DevExpress/testcafe/issues).
 However, to save some time, please search through the existing issues to see if the problem has already been reported or addressed.
```
Comment (dirk-pieterse) [inline CONTRIBUTING.md]: "However, to save some time, please"?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,33 +1,35 @@
...
 
 When you create a new issue, template text is automatically added to its body. To help us understand the issue you're describing, be sure to fill in all sections in this template. We process issues with insufficient details after all the others, which may take significant time without any guarantees.
```
Comment (dirk-pieterse) [inline CONTRIBUTING.md]: issue, **the** template text

Comment (dirk-pieterse) [reply CONTRIBUTING.md]: You should complete all the sections in this template to help us understand the issue you are describing. Missing information could delay the processing time.

File: CONTRIBUTING.md (hunk 2 above)
Comment (dirk-pieterse) [reply CONTRIBUTING.md]: We recommend searching through the existing issues to see if the problem has already been reported or addressed.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -41,7 +43,7 @@ Please follow the steps below when submitting your code.
 
 5. Fork TestCafe and create a branch in your fork. Name this branch with an issue number, e.g. `gh852`, `gh853`.
```
Comment (dirk-pieterse) [inline CONTRIBUTING.md]: 'e.g.' -> for example,

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,33 +1,35 @@
...
 
 When you create a new issue, template text is automatically added to its body. To help us understand the issue you're describing, be sure to fill in all sections in this template. We process issues with insufficient details after all the others, which may take significant time without any guarantees.
 
 ## Code Contribution
 
-Please follow the steps below when submitting your code.
+Follow the steps below when submitting your code.
 
 1. Search the [list of issues](https://github.com/DevExpress/testcafe/issues) to see if there
   is an issue for the bug or feature you are going to work on. If you do not find one, please create your own.
```
Comment (dirk-pieterse) [inline CONTRIBUTING.md]: "Search the [list of issues](https://github.com/DevExpress/testcafe/issues) to see if there
  is an issue for the bug or feature you are going to work on. If you do not find one, please create your own." -> Search the [list of issues](https://github.com/DevExpress/testcafe/issues) to see if there is an issue for the bug or feature you are going to work on or create a new one.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -61,11 +63,73 @@ Please follow the steps below when submitting your code.
...
+
+### Documentation for the Current Functionality
+
+If you want to fix a bug in the current documentation or make an enhancement that relates to the existing functionality, follow the instructions below.
+
+1. Fork TestCafe and create a branch in your fork. Name this branch with an issue number followed by the `docs` postfix, e.g. `gh852-docs`.
+
+2. Commit your changes into the branch.
+
+    > Note that links in the documentation should point to `.md` files in the repository, so that the documentation is browsable on GitHub. When the website is built, all links are automatically modified to point to HTML pages.
```
Comment (dirk-pieterse) [inline CONTRIBUTING.md]: to point to HTML pages -> to HTML links?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -61,11 +63,73 @@ Please follow the steps below when submitting your code.
...
+    gulp test-website
+    ```
+
+5. Push changes to your fork.
+
+6. Submit a pull request. Documentation pull requests should have the `[docs]` prefix in their title. This ensures that documentation tests are triggered against these pull requests.
+
+### Documentation for New Features
+
+Documentation for new features is developed in a separate branch `new-docs`.
```
Comment (dirk-pieterse) [inline CONTRIBUTING.md]: Why "is developed in"?

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -61,11 +63,73 @@ Please follow the steps below when submitting your code.
...
+    git checkout new-docs
+    ```
+
+3. Set the local branch to track the remote `new-docs` branch.
+
+    ```sh
+    git branch -u upstream/new-docs
+    ```
+
+4. Create a new branch on top of `new-docs`.
```
Comment (dirk-pieterse) [inline CONTRIBUTING.md]: Why 'on top of'?

**Pull request merged**